RubeesEat.IntegrationTests/SeleniumTests/UserHomepageSeleniumTests.cs

```diff
@@ -22,14 +22,6 @@ public void GetDefaultBalanceChanges()
         Assert.That(balanceChanges[2].Amount, Is.EqualTo("-800,00 \u20ac"));
         Assert.That(balanceChanges[2].Date, Is.EqualTo("Sunday, April 28, 2024"));
         Assert.That(balanceChanges[2].Description, Is.EqualTo("Lunch with default user in 2024"));
-
-        Assert.That(balanceChanges[3].Amount, Is.EqualTo("-800,00 \u20ac"));
-        Assert.That(balanceChanges[3].Date, Is.EqualTo("Friday, April 28, 2023"));
-        Assert.That(balanceChanges[3].Description, Is.EqualTo("Lunch with default user in 2023"));
-
-        Assert.That(balanceChanges[4].Amount, Is.EqualTo("-800,00 \u20ac"));
-        Assert.That(balanceChanges[4].Date, Is.EqualTo("Thursday, April 28, 2022"));
-        Assert.That(balanceChanges[4].Description, Is.EqualTo("Lunch with default user 2022"));
```

Comment:
For context: now that we have a page for showing bills, the front page only shows the 3 most recent bills, instead of 5